### PR TITLE
KH-232-fix: Add translations for CC form test sections' labels

### PR DESCRIPTION
**Pull request by kazlaw**
Link to ticket [here](https://mekomsolutions.atlassian.net/browse/KH-232)

### CODE REVIEW

**Comment by kazlaw** [comment]
@kdaud, kindly reference the video below for your testing and review

[232-fix.webm](https://github.com/openmrs/ozone-distro-cambodia/assets/36139599/c0985074-44f2-41bc-a3da-581342bdad0d)

File: configs/openmrs_config/ampathforms/cervical_cancer_screening.json [hunk 1]
```diff
@@ -764,10 +764,10 @@
 					},
 					{
 					"isExpanded": "true",
-					"label": "Cervical Biopsis",
```
Comment (kdaud) [inline configs/openmrs_config/ampathforms/cervical_cancer_screening.json:767]: @kazlaw, why are we changing the label name?

Comment (kazlaw) [reply configs/openmrs_config/ampathforms/cervical_cancer_screening.json:767]: Its a more accurate label

Comment (kazlaw) [reply configs/openmrs_config/ampathforms/cervical_cancer_screening.json:767]: Link to cell [here](https://docs.google.com/spreadsheets/d/13PHkzorfTmYRjuIJv3fCml3zBAuDba6Mgdpu4H7cAg8/edit#gid=0&range=225:225)

**Review by kdaud — APPROVED** [review]

**Pull request merged**